app/models.py

```diff
@@ -550,3 +557,21 @@ def bulk_lookup():
                 db.session.commit()
             except IntegrityError:
                 db.session.rollback()
+
+class Leaders(db.Model):
```

Comment:
You need to lose migration 77b94550f2ea or drop the model tables, don't you? I get:
```
$ ./manage.py deploy
...
sqlalchemy.exc.ProgrammingError: (psycopg2.ProgrammingError) relation "priority" already exists
 [SQL: "create view priority
```